content/doc/book/system-administration/security.adoc

```diff
@@ -87,3 +87,9 @@ When this happens, you can fix this by the following steps:
   access to the system.
 
 If this is still not working, trying renaming or deleting `config.xml`.
+
+[IMPORTANT] 
+.Using configuration as code
+====
+In case you are configuring your security realm / authorization using JCasc or Groovy init scripts, you will also need to deactivate JCasc or your Groovy init scripts so that they do not override the security part of the `config.xml`.
```

Comment:
bit vague really, we often have people not sure how to turn it off, probably better to provide example configuration or actually tell them how to turn it off...

Reply:
done